### PR TITLE
Russian language for multilaguage system

### PR DESCRIPTION
**Pull request by AlexTriada**
everything according to plan, as agreed

### CODE REVIEW

**Comment by Toakan** [comment]
Looks good to me, @AlexTriada just for my curiosity, can you screenshot what this looks like for you now?

File: A3-Antistasi/Stringtable.xml [hunk 1]
```diff
@@ -11,278 +11,273 @@
...
     <Container name="dialog_unit_recruit">
       <Key ID="STR_antistasi_dialogs_unit_recruit_frame_text">
         <Original>Recruitment Options</Original>
         <English>Recruitment Options</English>
+        <Russian>Обучить Добровольца</Russian>
       </Key>
       <Key ID="STR_antistasi_dialogs_unit_recruit_militiaman_text">
         <Original>Recruit Militiaman</Original>
         <English>Recruit Militiaman</English>
+        <Russian>+1 Новобранец</Russian>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: Can this not be `verbovat` or `вербовать`? Instead of `+`

File: A3-Antistasi/Stringtable.xml [hunk 2]
```diff
@@ -11,278 +11,273 @@
...
       </Key>
       <Key ID="STR_antistasi_dialogs_unit_recruit_mg_text">
         <Original>Recruit Autorifleman</Original>
         <English>Recruit Autorifleman</English>
+        <Russian>+1 Пулемётчик</Russian>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: Can this not be `verbovat` or `вербовать`? Instead of `+`

File: A3-Antistasi/Stringtable.xml [hunk 3]
```diff
@@ -11,278 +11,273 @@
...
       </Key>
       <Key ID="STR_antistasi_dialogs_unit_recruit_medic_text">
         <Original>Recruit Medic</Original>
         <English>Recruit Medic</English>
+        <Russian>+1 Медик</Russian>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: Can this not be `verbovat` or `вербовать`? Instead of `+`

File: A3-Antistasi/Stringtable.xml [hunk 4]
```diff
@@ -11,278 +11,273 @@
...
       </Key>
       <Key ID="STR_antistasi_dialogs_unit_recruit_engineer_text">
         <Original>Recruit Engineer</Original>
         <English>Recruit Engineer</English>
+        <Russian>+1 Инженер</Russian>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: Can this not be `verbovat` or `вербовать`? Instead of `+`

File: A3-Antistasi/Stringtable.xml [hunk 5]
```diff
@@ -11,278 +11,273 @@
...
       </Key>
       <Key ID="STR_antistasi_dialogs_unit_recruit_explosive_text">
         <Original>Recruit Bomb Specialist</Original>
         <English>Recruit Bomb Specialist</English>
+        <Russian>+1 Подрывник</Russian>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: Can this not be `verbovat` or `вербовать`? Instead of `+`

File: A3-Antistasi/Stringtable.xml [hunk 6]
```diff
@@ -11,278 +11,273 @@
...
       </Key>
       <Key ID="STR_antistasi_dialogs_unit_recruit_grenadier_text">
         <Original>Recruit Grenadier</Original>
         <English>Recruit Grenadier</English>
+        <Russian>+1 Стрелок с ГП</Russian>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: Can this not be `verbovat` or `вербовать`? Instead of `+`

File: A3-Antistasi/Stringtable.xml [hunk 7]
```diff
@@ -11,278 +11,273 @@
...
       </Key>
       <Key ID="STR_antistasi_dialogs_unit_recruit_marksman_text">
         <Original>Recruit Marksman</Original>
         <English>Recruit Marksman</English>
+        <Russian>+1 Снайпер</Russian>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: Can this not be `verbovat` or `вербовать`? Instead of `+`

File: A3-Antistasi/Stringtable.xml [hunk 8]
```diff
@@ -11,278 +11,273 @@
...
       </Key>
       <Key ID="STR_antistasi_dialogs_unit_recruit_antitank_text">
         <Original>Recruit Antitank Solider</Original>
         <English>Recruit Antitank Solider</English>
+        <Russian>+1 Гранатомётчик</Russian>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: Can this not be `verbovat` or `вербовать`? Instead of `+`

Comment (AlexTriada) [reply A3-Antistasi/Stringtable.xml]: "Вербовать" takes up more space, and in the Russian language is more often used in the sense of persuading to switch to one’s side

Comment (AlexTriada) [reply A3-Antistasi/Stringtable.xml]: The word "взять" is more suitable, but it sounds ugly

Comment (AlexTriada) [reply A3-Antistasi/Stringtable.xml]: In the Antistasi 1.9 it was only "Гранатомётчик" word

Comment (Toakan) [reply A3-Antistasi/Stringtable.xml]: I'll let you decide the best way to do this, if you think it's suitable I'll accept it?

**Comment by AlexTriada** [comment]
> Can this be changed?

Let's leave it this way, time will tell how best)

**Comment by AlexTriada** [comment]
> Looks good to me, @AlexTriada just for my curiosity, can you screenshot what this looks like in game?

Ok i will do, tomorrow

**Pull request merged**